app/src/hooks/useLayer1Balance.ts

```diff
@@ -0,0 +1,49 @@
+import { useCurrentAccount } from '@iota/dapp-kit';
+import { useBalance as useBalanceL1 } from './useBalance';
```

Comment:
Consider renaming the alias to `useFetchL1Balance` or something. The name sounds too similar to this hook (`useLayer1Balance`). Or rename the hook defined in this file to `useL1AvailableBalance` or similar, just to try to convey the difference between what these two do.
Same in L2 version.